webapp/src/main/webapp/news-list-view/components/views/NewsListTemplateViewItem.vue

```diff
@@ -25,11 +25,11 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
     <div class="article-item-content">
       <span
         v-if="showArticleTitle"
-        class="text-color text-body-2"
+        class="text-body-1 black--text"
         :class="extraClass">{{ item.title }}</span>
       <span
         v-if="showArticleSummary"
-        class="text-color text-body-2"
+        class="text-body-1 black--text"
```

Comment:
shouldn't we need to use primary text color, not black one ?

Reply:
in the task " article title style must be the same as displayed documents titles in Documents widget" which is in black

Reply:
I think we need to use this color https://github.com/Meeds-io/platform-ui/blob/develop/platform-ui-skin/src/main/webapp/skin/less/helpers.less#L40
could you check with Ikbel ?

Reply:
done!